recipe/meta.yaml

```diff
@@ -17,71 +13,42 @@ source:
     - 0001-use-torch.tensor-to-create-a-tensor-with-initializer.patch
 
 build:
-  number: 3
-  skip: True  # [osx or win]
-  # as of pytorch 1.13, conda-forge only builds for CUDA 11.2+, see
-  # https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/3491
-  skip: true  # [cuda_compiler_version in ("10.2", "11.0", "11.1")]
+  number: 4
+  skip: true  # [not linux or cuda_compiler == "None"]
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+  script_env:
+    - TORCH_CUDA_ARCH_LIST=3.5;5.0;6.0;6.1;7.0;7.5;8.0;8.6+PTX  # [cuda_compiler_version == "11.2"]
+  script:
+    - python -m pip install . -vv
 
-outputs:
-  - name: nvidia-apex-proc
-    version: {{ proc_version }}
-    build:
-      string: {{ proc_type }}
-    test:
-      commands:
-        - exit 0
-    about:
-      home: https://github.com/conda-forge/nvidia-apex-feedstock
-      license: BSD-3-Clause
-      license_family: BSD
-      summary: A meta-package to select CPU or GPU nvidia-apex build.
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+  host:
+    - python
+    - pytorch
+    - pytorch =*=cuda*
+    - setuptools
+    - pip
+  run:
+    - python
+    - cxxfilt
+    - tqdm
+    - numpy
+    - PyYAML
+    - pytest
+  run_constrained:
+    - pytorch =*=cuda*
 
-  - name: nvidia-apex
-    version: {{ version }}
```

Comment:
Are you absolutely convinced that we'll never want or need another output on this feedstock?

Because if the answer is not "yes", then I think we should just keep the output, and avoid all this churn.

Reply:
Yep. It was only here for the CPU-builds, which are gone

Reply:
I'd don't think there's a meaningful difference in terms of simplicity between one output and none, except that we mess up the diff and the history.

That said, I only signed up for this feedstock because it was effectively unmaintained, so I don't care all that much. Feel free to go either way

Reply:
We could [exclude relevant commits from `git blame`]( https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view )

Reply:
I've made my preference (and indifference) clear. Proceed however you see fit

Reply:
Ok was just trying to address the concern over git history. If that's not relevant, happy to proceed as-is. We can always revisit